### PR TITLE
Hotfix dat writers

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable) (N/A)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable) (N/A)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <http://link-to-pr.com>`_) Added feature which does something
- (`#XX <http://link-to-pr.com>`_) Fixed bug identified in (`#XX <http://link-to-issue.com>`_)
```

### CODE REVIEW

File: pymagicc/io.py [hunk 1]
```diff
@@ -2864,116 +2516,363 @@ def rename_col(x):
...
-            "HCFC142B_CONC",
-            "HALON1211_CONC",
-            "HALON1202_CONC",
-            "HALON1301_CONC",
-            "HALON2402_CONC",
-            "CH3BR_CONC",
-            "CH3CL_CONC",
-        ]
+    def _get_col_order_rename_col(self):
+        if self._filepath.endswith("_RADFORCING.DAT"):
```
Comment (lewisjared) [inline pymagicc/io.py:2523]: Are the following the same except for the _RF/_EFFRF?

File: pymagicc/io.py [hunk 2]
```diff
@@ -2363,69 +2351,26 @@ def _write_namelist_radforcing(self, output):
...
-            + len(line_after_nml.split(self._newline_char))
-            + number_col_headers
-        )
-        nml["THISFILE_SPECIFICATIONS"]["THISFILE_UNITS"] = "SEE ROW 37"
-        nml["THISFILE_SPECIFICATIONS"]["THISFILE_DATTYPE"] = "RCPDAT"
-
-        nml.uppercase = True
-        nml._writestream(output)
-        output.write(line_after_nml)
+        if "_RADFORCING.DAT" in self._filepath:
```
Comment (lewisjared) [inline pymagicc/io.py:2356]: self._filepath.endswith was used previously

**Comment by lewisjared** [comment]
Other than that LGTM

File: pymagicc/io.py (hunk 1 above)
Comment (znicholls) [reply pymagicc/io.py]: resolved by e5792b74b7bc7727031ffa4a1d54c0adc2d8cd96

**Pull request merged**